api/models/topics-models.js

```diff
@@ -6,4 +6,22 @@ exports.selectTopics = () => {
     return db.query(queryString).then((result) => {
         return result.rows
     })
-}
\ No newline at end of file
+}
+
+exports.selectArticlesById = (article_id) => {
+    let queryString = `
+    SELECT * FROM articles
+    `
+    if (!Number(article_id)){
```

Comment:
PSQL will handle this, you do not need to promise reject. The database is specifically looking for a number, if you do not provide one you will get a 22P02 error which you should be handling.